packages/storybook/src/stories/ino-table/ino-table.stories.ts

```diff
@@ -42,15 +42,15 @@ export default {
   },
 };
 
-const tableContent = html`
-    <tr slot="header-row"><ino-table-header-cell column-id="id" sort-start="asc"  label="ID"></ino-table-header-cell>
+const tableContent = 
+    html`<tr slot="header-row"></tr>
```

Comment:
Is this right, that the `tr` does not inclues the `ino-table-header-cell` elements

Reply:
if it includes the `ino-table-header-cell` elements the big gap appears... i couldn't find another way to refactor the template so that the gap disappears.  
It doesn't change the appearance of the table so i left it like that.